hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestOmMetadataManager.java

```diff
@@ -689,12 +690,17 @@ public void testListSnapshot() throws Exception {
     }
 
     //Test listing snapshots with no volume name.
-    Assert.assertThrows(OMException.class, () -> omMetadataManager.listSnapshot(
-        null, null));
+    Assert.assertThrows(OMException.class,
+        () -> omMetadataManager.listSnapshot(null, null));
 
     //Test listing snapshots with no bucket name.
-    Assert.assertThrows(OMException.class, () -> omMetadataManager.listSnapshot(
-        vol1, null));
+    Assert.assertThrows(OMException.class,
+        () -> omMetadataManager.listSnapshot(vol1, null));
+
+    //Test listing snapshots with non-existent bucket.
+
+    Assert.assertThrows(OMException.class,
+        () -> omMetadataManager.listSnapshot(vol1, nonexistentBucket));
```

Comment:
nit: we can also verify the error code here. 
```suggestion
            OzoneTestUtils.expectOmException(BUCKET_NOT_FOUND,
        () -> omMetadataManager.listSnapshot(vol1, nonexistentBucket));
```

Reply:
Done.